packages/core/aqua-to-js/package.json

```diff
@@ -21,9 +21,11 @@
     "@fluencelabs/aqua-api": "0.12.0",
     "@fluencelabs/aqua-lib": "0.7.3",
     "@fluencelabs/interfaces": "workspace:*",
+    "@fluencelabs/js-client": "0.4.3",
     "@fluencelabs/registry": "0.8.7",
     "@fluencelabs/spell": "0.5.20",
     "@fluencelabs/trust-graph": "0.4.7",
```

Comment:
does it need all of these? why?

Reply:
I use `@fluencelabs/registry` in test. Removed the rest